starky/src/lib.rs

```diff
@@ -1,14 +1,333 @@
+//! A FRI-based STARK implementation over the Goldilocks field, with support
+//! for recursive proof verification through the plonky2 SNARK backend.
+//!
+//! This library is intended to provide all the necessary tools to prove,
+//! verify, and recursively verify STARK statements. While the library
+//! is tailored for a system with a single STARK, it also is flexible
```

Comment:
Starky on its own is tailored for single-STARK system. There is no clear API for building multi-STARK systems yet, like the one on the EVM side, although the pieces are there so that people can build them.